src/plugins/blocksmith/components/image-block-modal/CustomizeOptions.jsx

```diff
@@ -49,107 +57,155 @@ const CustomizeOptions = ({
-  <div className="modal-body-blocksmith">
-    <ImageBlockPreview className="my-4" block={block} />
-    {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
-      <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
-    )}
-    <FormGroup>
-      <ImageAssetPicker
-        multiAssetErrorMessage="Invalid Action: Trying to assign multiple Gallery Block Poster images."
-        isImageSelected={!!selectedImage}
-        isModalOpen={isAssetPickerModalOpen}
-        isDisabled={false}
-        label="Select an image"
-        node={node}
-        onClearImage={handleClearImage}
-        onSelectImage={handleSelectImage}
-        onToggleAssetPickerModal={handleToggleAssetPickerModal}
-      />
-    </FormGroup>
-    <div style={{ maxWidth: '350px', margin: '0 auto' }}>
+}) => {
+  const hasThemeableMixin = () => node.schema().hasMixin('triniti:common:mixin:themeable');
+
+  return (
+    <div className="modal-body-blocksmith">
+      <ImageBlockPreview className="my-4" block={block} />
+      {hasCaption && (block.has('caption') || selectedImage.has('description')) && (
+        <p>Caption: {block.get('caption') || selectedImage.get('description')}</p>
+      )}
       <FormGroup>
-        <Label>Aspect Ratio</Label>
-        <Select
-          onChange={handleChangeAspectRatio}
-          value={!aspectRatio.value ? null : {
-            label: aspectRatio.value.replace('by', ' by '),
-            value: aspectRatio.value,
-          }}
-          options={aspectRatioOptions}
+        <ImageAssetPicker
+          multiAssetErrorMessage="Invalid Action: Can only select one image."
+          isImageSelected={!!selectedImage}
+          isModalOpen={isAssetPickerModalOpen}
+          isDisabled={false}
+          label="Select an image"
+          node={node}
+          onClearImage={handleClearImage}
+          onSelectImage={handleSelectImage}
+          onToggleAssetPickerModal={handleToggleAssetPickerModal}
         />
       </FormGroup>
-      <FormGroup className="d-flex mb-2">
-        <FormGroup check>
-          <Label check>
-            <Checkbox size="sd" id="isNsfw" checked={isNsfw} onChange={handleChangeCheckbox} />
-            is nsfw
-          </Label>
+      <div className="custom-options-container">
+        <FormGroup>
+          <Label>Aspect Ratio</Label>
+          <Select
+            onChange={handleChangeAspectRatio}
+            value={!aspectRatio.value ? null : {
+              label: aspectRatio.value.replace('by', ' by '),
+              value: aspectRatio.value,
+            }}
+            options={aspectRatioOptions}
+          />
         </FormGroup>
-      </FormGroup>
-      <FormGroup className="d-flex align-items-start mb-2">
-        <FormGroup check className="mr-2 pb-1">
-          <Label check>
-            <Checkbox size="sd" id="hasCaption" checked={hasCaption} onChange={handleChangeCheckbox} />
-            caption
+
+        {hasThemeableMixin()
```

Comment:
@jalopez13 put the mixin check right here `node.schema().hasMixin('triniti:common:mixin:themeable')`.  adding a function creation for every render is not worth the extra function